docs-developer/loading-in-profiles.md

```diff
@@ -102,4 +102,4 @@ Firefox loads the profiles directly into the front-end through a WebChannel mech
 
 > `https://profiler.firefox.com/from-file/`
 
-When you're on [the home page](https://profiler.firefox.com) files can be loaded by either dragging over the profiler.firefox.com client, or using the file upload input.
+When you're on [the home page](https://profiler.firefox.com) files can be loaded by either dragging over the profiler.firefox.com client, or using the file upload input. On Linux, the provided [fp.sh](../bin/fp.sh) script can load files.
```

Comment:
I feel like this addition should be in another paragraph though. Indeed we're really talking about the `from-file` datasource here.

What about moving this doc in the URL part above, below the part `Here is an example profile server written in Node.js:`:


> Note that if you have a copy of the project locally, you can simply add the file path as a parameter to `yarn start` and a server serving this file will be spawned for you. On Linux, the provided [launch-fp.sh](../bin/launch-fp.sh) script can do this for you as well, from any working directory.

Reply:
Ok lgtm with some minor editing (file -> profile, and removed 'simply' because alex complained) fe0bbab29c07ae25a948b2e412e7cdcdcbba6a4b fb1b549e23971d34a736658468ed837944db47ca

> Indeed we're really talking about the from-file datasource here.

The document looked like it was for users that are also developers rather than Firefox Profiler developers specifically.